test/cases/api/common/vsphere.sh

```diff
@@ -215,3 +215,34 @@ function verifyInVSphere() {
 
     greenprint "✅ Successfully verified VSphere image with cloud-init"
 }
+
+function cleanupVSphere() {
+    # since this function can be called at any time, ensure that we don't expand unbound variables
```

Comment:
If the variables are unbound, should we instead return from the function and not end up running `$GOVC_CMD vm.destroy ...` with an empty var?

Reply:
We could, but we don't do it in any of the cleanup functions. It is cleanup, we do not care too much TBH. I'm happy to add the check, but we should be consistent and add it into all existing cleanup functions.

Reply:
It's fine.  Maybe we could do it for all another time.